stable-diffusion/stable-diffusion-inpainting-trt/config.yaml

```diff
@@ -0,0 +1,26 @@
+environment_variables: {}
+external_package_dirs: []
+model_metadata: {}
+model_name: Stable Diffusion Inpainting TRT
+python_version: py310
+requirements:
+  - accelerate
```

Comment:
can we put this in a requirements.txt and use `requirements_file: requirements.txt` instead? please make sure all versions are pinned.